README.md

```diff
@@ -41,39 +41,41 @@ This tool expects to run in an environment with the following attributes:
 
 Requesting a code review:
 
-    git appraise request
+    git review request
 
 Pushing code reviews to a remote:
 
-    git appraise push [<remote>]
+    git review push [<remote>]
 
 Pulling code reviews from a remote:
 
-    git appraise pull [<remote>]
+    git review pull [<remote>]
 
 Listing open code reviews:
 
-    git appraise list
+    git review
+    git review --list
```

Comment:
my suggestions are in line with the builtin `git branch` and `git tag` for instance

NOTE: should also support the short `-l`